include/circt/Dialect/MSFT/MSFTPDOps.td

```diff
@@ -183,3 +186,17 @@ def DynamicInstanceVerbatimAttrOp : MSFTOp<
     }
   }];
 }
+
+def PDInstanceHierarchyCallOp : MSFTOp<"instance.hierarchy.call", [
```

Comment:
nit: I don't like naming this "call" since that implies that it's necessarily tied to a particular language construct, when it could (possibly) be output some other way.